pkg/providers/tinkerbell/hardware/multi.go

```diff
@@ -0,0 +1,25 @@
+package hardware
```

Comment:
File move, github hasn't picked it up because of the rename from `TeeWriter` to `multiWriter`.